config/sync/user.role.contributor.yml

```diff
@@ -7,15 +7,32 @@ label: Contributor
 weight: -4
 is_admin: null
 permissions:
```

Comment:
- This role can only edit their own content. The google doc does not mention if this is the desire.
- They currently cant view or revert revisions
- They can delete their own content but cant unpublish
- They can edit the main menu & delete/create/rearrange links